server/server.go

```diff
@@ -1667,8 +1667,7 @@ func (s *Server) UnmarkSnapshotRecovering(ctx context.Context) error {
 
 // GetServicePrimaryAddr returns the primary address for a given service.
 func (s *Server) GetServicePrimaryAddr(ctx context.Context, serviceName string) (bool, string, error) {
-	// TODO: replace default group name after we make a decision.
-	key := path.Join("/ms/0", serviceName, fmt.Sprintf("%05d", 0), "primary")
+	key := fmt.Sprintf("/ms/%d/%s/%s/%s", s.clusterID, serviceName, fmt.Sprintf("%05d", 0), "primary")
```

Comment:
Will this break compatibility?

Reply:
It's not used for now, so I think there is no compatibility issue.